source/js/components/join/join.jsx

```diff
@@ -48,6 +48,18 @@ export default class JoinUs extends React.Component {
     );
   }
 
+  /**
```

Comment:
Copied from `petition.jsx`: https://github.com/mozilla/foundation.mozilla.org/blob/master/source/js/components/petition/petition.jsx#L262-L272

Reply:
interesting...wonder why we didn't use dummy form field validation i.e. something like:

```js
function isValidEmail(email) {
  const el = document.createElement('input');
  el.type = `email`;
  el.required = true;
  el.value = email;
  return el.validity.valid;
}
```

either way, this is still fine for majority emails.

Reply:
I'll mention this in this ticket: #3005